.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
Tests should be run on each `push` and `pull_request`, not just on master.

Only semantic release should be restricted to being run on master.